packages/adminrouter/extra/src/test-harness/modules/generic_test_code/common.py

```diff
@@ -71,6 +74,8 @@ def generic_upstream_headers_verify_test(ar, auth_header, path, assert_headers=N
         path (str): path for which request should be made
         assert_headers (dict): additional headers to test where key is the
             asserted header name and value is expected value
+        assert_header_absent (dict): headers that *MUST NOT* be present in the
```

Comment:
re: typo - fixed!
re: type hints - Two things:
* google style guideline is not very specific regarding the types: `The description should mention required type(s) and the meaning of the argument.` So for me it's unclear if in the docstring I should mention that the variable can be `None` or is it implied. 
* I agree that the solution to this ambiguity would be type hints and I am open to implementing them, but I do not see it happening in 1.10. We can create a Jira ("Implement type hints for *whole* AR test harness"), assign it to me and I will see what we can do :)